src/pybind/mgr/dashboard/tox.ini

```
@@ -105,6 +105,7 @@ deps =
     {[base-lint]deps}
 commands =
     flake8
+    flake8 --config=tox.ini ../../../../qa/tasks/mgr/dashboard
```

Could you also add `isort`?
```suggestion
    flake8 --config=tox.ini ../../../../qa/tasks/mgr/dashboard
    isort ../../../../qa/tasks/mgr/dashboard --check
```

It would also be good to add the correspondent fix commands:
```
autopep8 {[autopep8]addopts} ../../../../qa/tasks/mgr/dashboard
isort ../../../../qa/tasks/mgr/dashboard
```

---

And maybe add it to `pylint` as well: ... ?

```
[pylint]
# Allow similarity/code duplication detection
jobs = 1
dirs = . controllers plugins services tests ../../../../qa/tasks/mgr/dashboard
addopts = -rn --rcfile=.pylintrc --jobs={[pylint]jobs}
```

---

i think the next step is to integrate this check with https://github.com/ceph/cbt/pull/226 .